account_statement_import_camt/models/parser.py

```diff
@@ -330,12 +329,14 @@ def parse_entry(self, ns, node):
         details_nodes = node.xpath("./ns:NtryDtls/ns:TxDtls", namespaces={"ns": ns})
         if len(details_nodes) == 0:
             self.generate_narration(transaction)
+            self.amend_transaction(transaction)
```

Comment:
please move this before the call to `generate_narration`, at this point the field is already flattened to a string

Reply:
@hbrunn , done!

Reply:
thanks